base_station/gui/src/components/LCMEcho.vue

```diff
@@ -139,6 +144,7 @@
           {'topic': '/target_list', 'type': 'TargetList'},
           {'topic': '/temperature', 'type': 'Temperature'},
           {'topic': '/tennis_ball', 'type': 'TennisBall'},
+          {'topic': '/thermistor_data', 'type': 'ThermistorData'},
           {'topic': '/imu', 'type': 'IMU'},
           {'topic': '/gps', 'type': 'GPS'},
           {'topic': '/microscope', 'type': 'Microscope'},
```

Comment:
is spectral also here?

Reply:
No, but this is just the LCM Echo page. It was added into the LCM send page for testing, but having it here is not really necessary for function.